uint8 id1
uint8 id2
uint8 id3
uint8 id4
uint8 id5
uint8 id6
uint8 id7
uint8 id8
uint8 id9
uint8 id10
uint8 id11
uint8 id12
---
int32 position1
int32 position2
int32 position3
int32 position4
int32 position5
int32 position6
int32 position7
int32 position8
int32 position9
int32 position10
int32 position11
int32 position12
